tpl/tplimpl/template_funcs.go

```diff
@@ -474,6 +474,11 @@ func (ic *imageHandler) config(path interface{}) (image.Config, error) {
 
 // in returns whether v is in the set l.  l may be an array or slice.
 func in(l interface{}, v interface{}) bool {
+
+	if nil == v {
```

Comment:
Not needed are [Yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions).